add-on/src/popup/browser-action/gateway-status.js

```diff
@@ -27,31 +27,27 @@ module.exports = function gatewayStatus ({
   const api = ipfsApiUrl && ipfsNodeType === 'embedded' ? 'js-ipfs' : ipfsApiUrl
   return html`
     <ul class="fade-in list mv0 pv2 ph3 white">
-  ${statusEntry({
-    label: 'panel_statusGatewayAddress',
-    value: gatewayAddress,
-    check: gatewayAddress,
-    itemClass: 'mb1'
-  })}
-  ${statusEntry({
-    label: 'panel_statusApiAddress',
-    value: api,
-    check: gatewayVersion,
-    itemClass: 'mb1'
-  })}
-  ${statusEntry({
-    label: 'panel_statusGatewayVersion',
-    value: gatewayVersion,
-    check: gatewayVersion,
-    itemClass: 'mb1'
-  })}
-  ${statusEntry({
-    label: 'panel_statusSwarmPeers',
-    labelLegend: 'panel_statusSwarmPeersTitle',
-    value: swarmPeers,
-    check: swarmPeers,
-    valueClass: 'fw9'
-  })}
+    ${statusEntry({
+      label: 'panel_statusSwarmPeers',
+      labelLegend: 'panel_statusSwarmPeersTitle',
+      value: swarmPeers,
```

Comment:
Random idea: would it be useful if we make this peer count clickable with `cursor: help` to open `#/peers` in WebUI?

items below could open Preferences and scroll to relevant sections (`/dist/options/options.html#gateway`  and `/dist/options/options.html#api`)

Reply:
Do you mind if we defer on that until we see how the new design is received? Once I got a help cursor on each of those elements, the clickable items were really close to each other spatially -- it's fairly easy to click on the wrong thing.